### PR TITLE
Add additional valid parameters to linting rules

### PR DESCRIPTION
**Pull request by gingeranyhow**

### CODE REVIEW

**Comment by gingeranyhow** [comment]
@awmcclain - LMK if you want any changes here!

**Comment by gingeranyhow** [comment]
Also - I'm assuming you can squash merge, but if not I can make this a single clean commit.

File: app/linter/ink-linter.js [hunk 1]
```diff
@@ -57,8 +124,28 @@ let storyTags = {
   linter: hasStoryTagError,
   needsParam: true,
   validParams: {
-    cutCamera: ['linapov', 'paxleft', 'paxright', 'dashcam', 'hoodcam', 'linafront', 'city'],
-    linaHands: ['checkwatch']
+    cutCamera: [
+      'linapov',
+      'LinaFrontClose',
```
Comment (awmcclain) [inline app/linter/ink-linter.js]: Should these be all lower for the linter?

**Review by awmcclain — COMMENTED** [review]
Thank you thank you!!!!

**Comment by gingeranyhow** [comment]
yea, they should. I had originally made a change that mapped through the
parameters and also put them lower-case, but decided to pull it out and
didn't update these. I'll test and put a new commit up in the morning.

File: app/linter/ink-linter.js (hunk 1 above)
Comment (gingeranyhow) [reply app/linter/ink-linter.js]: @awmcclain  - fixed!

**Pull request merged**